packages/core/src/utils/ConstantsUtil.ts

```diff
@@ -61,7 +61,101 @@ export const ConstantsUtil = {
     Base: 'base'
   },
 
-  WC_COINBASE_ONRAMP_APP_ID: 'bf18c88d-495a-463b-b249-0b9d3656cf5e'
+  WC_COINBASE_ONRAMP_APP_ID: 'bf18c88d-495a-463b-b249-0b9d3656cf5e',
+
+  SUGGESTED_TOKENS: [
+    'ETH',
+    'UNI',
+    '1INCH',
+    'AAVE',
+    'SOL',
+    'ADA',
+    'AVAX',
+    'DOT',
+    'LINK',
+    'NITRO',
+    'GAIA',
+    'MILK',
+    'TRX',
+    'NEAR',
+    'GNO',
+    'WBTC',
+    'DAI',
+    'WETH',
+    'USDC',
+    'USDT',
+    'ARB',
+    'BAL',
+    'BICO',
+    'CRV',
+    'ENS',
+    'MATIC',
+    'OP'
+  ],
+
+  POPULAR_TOKENS: [
+    'ETH',
```

Comment:
Some list of tokens to show for the demo purpose. Waiting Lit's virtualize SSR support for listing all possible tokens in the performant way.

- https://github.com/lit/lit/pull/4564

Reply:
same

Reply:
Created a post merge ticket for this: https://linear.app/walletconnect/issue/W3M-258/post-merge-improvements-for-convert